lib/tokenize-arg-string.ts

```diff
@@ -1,9 +1,15 @@
 // take an un-split argv string and tokenize it.
-module.exports = function (argString) {
+export function tokenizeArgString (argString: string | any[]): string[] {
```

Comment:
perhaps call this `maybeArgString`.

Reply:
I was just keeping with the existing argument name. It does keep a nice consistency with the function name though. I really don't have a strong preference either way. 

My thought on if changing the argument name to `maybeArgString` would be to update the first few lines like so:

```js
export function tokenizeArgString (maybeArgString: string | any[]): string[] {
  if (Array.isArray(maybeArgString)) {
    return maybeArgString.map(e => typeof e !== 'string' ? e + '' : e)
  }

  const argString = (maybeArgString as string).trim()
```

Another option would be to drop the type from name. It's just called `args` in parse when calling `tokenizeArgString(args)`. Then to keep the same type of naming consistency could change the function name to `tokenizeArgs`. This would be my personal preference.

Just let me know what y'all want and I'll update as needed.

Reply:
I would recommend not to rename functions or arguments, once again to limit code changes.